README.md

```diff
@@ -1,8 +1,38 @@
-# xcs-translator
+# XCS Translator
+
+XCS Translator is a conversion tools to help you upload an EXCEL file, it will then convert the uploaded file into JSON and also SQL. Enabling you to quickly upload EXCEL sheet into your database.
+
+First you need to define your excel template in `sheetconfig.json` file. All uploaded data will be validated against the template definition. You can also add format constraints for each column using [validate.js](https://validatejs.org/#validators) syntax.
+
+The uploaded files will be availabe in `storage` directory with following structure:
+
+```bash
+storage
+├── data1
+│   ├── archive
+│   │   ├── excel
+│   │   └── json
+│   ├── excel
+│   ├── failed
+│   │   ├── excel
+│   │   └── json
+│   └── json
+└── data2
+    ├── archive
+    │   ├── excel
+    │   └── json
+    ├── excel
+    ├── failed
+    │   ├── excel
+    │   └── json
+    └── json
+```
+
+While it's processed, the uploaded file resides under `data1` or `data2` (depends on which data is uploaded) inside `excel` directory for raw file, and `json` directory after it's converted to json. Then it will move to `archive` directory if no error happens and data is successfully inserted into database, or to `failed` directory if there is error. The failed files should be fixed manually and uploaded again.
```

Comment:
While being processed, your raw excel file will be stored in `data1/excel` or `data2/excel` directory. The json conversion result will be stored in `json` directory. After successfully inserted into targeted database, your raw excel file will be move to `archive` directory. If there are some exception occurred due to validation failure, the excel file will be move to `failed` directory. Any failed files if you want to re-upload, should be fix manually so all records can pass validation.

Reply:
Just few corrections:

- "After successfully inserted into targeted database" -> "After successfully inserted into database". I think no need to say targeted anymore.
- "your raw excel file will be move to archive directory" -> "your raw excel file will be moved to archive directory"
- "If there are some exception occurred due to validation failure" -> "If some exceptions occurred due to validation failure"
- "the excel file will be move to failed directory" -> "the excel file will be moved to failed directory"
- "Any failed files if you want to re-upload, should be fix manually so all records can pass validation" -> "Failed files should be fixed manually and re-uploaded to pass the validation."

Reply:
wording has been updated